### PR TITLE
Teleprods fixes

### PR DESCRIPTION
**Pull request by PKPenguin321**
by removing the ability to remove the crystal

it was only added in the first place since certain people worried that assistants would fuck up the teleporter roundstart and steal all the crystals and make them unrecoverable (which they could already do by just throwing the crystals but whatever) by crafting teleprods and i wanted to appease everybody

it's proved pretty much unmaintainable due to a combo of artificial bluespace crystals existing and UI crafting being a really bad system and not allowing me to set a var on the prod when it's created depending on what kind of crystal was used

Fixes https://github.com/tgstation/tgstation/issues/16888

Also changes some messages to match igniter stun prods

Fixes https://github.com/tgstation/tgstation/issues/15682

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -43,8 +43,8 @@
 #include "code\__DEFINES\reagents.dm"
 #include "code\__DEFINES\role_preferences.dm"
 #include "code\__DEFINES\say.dm"
-#include "code\__DEFINES\sight.dm"
 #include "code\__DEFINES\shuttles.dm"
+#include "code\__DEFINES\sight.dm"
```
Comment (PKPenguin321) [inline tgstation.dme]: wat

File: tgstation.dme [hunk 2]
```diff
@@ -43,8 +43,8 @@
 #include "code\__DEFINES\reagents.dm"
 #include "code\__DEFINES\role_preferences.dm"
 #include "code\__DEFINES\say.dm"
-#include "code\__DEFINES\sight.dm"
```
Comment (PKPenguin321) [inline tgstation.dme]: ????  git

**Comment by GunHog** [comment]
What if you had two kinds of prods? Artificial being inferior to the "natural" version?

File: tgstation.dme (hunk 2 above)
Comment (GunHog) [inline tgstation.dme]: You forgot close DM when you changed your branch, I would bet.

**Comment by PKPenguin321** [comment]
it would cause a lot of copied code and violate many code standards

**Comment by GunHog** [comment]
How? Make the weaker one a child of the base one. Variables different on the child. You do not need to duplicate anything except having similar recipes in the crafting menu.

**Comment by PKPenguin321** [comment]
the code for removing the crystal is embedded in the attack_self and it's such a pain in the ass man i dont know ive been hacking at this for maybe an hour or two now and it's not worth doing if you wanna try it be my guest

**Comment by Razharas** [comment]
>not allowing me to set a var on the prod when it's created depending on what kind of crystal was used

Thats false, all the parts used in construction are passed to the created object as a list argument in CheckParts proc, on the atom level they are added to the result's contents, so all you need to do is check the type of crystal in the teleprod's conents

**Comment by PKPenguin321** [comment]
you do it then sheesh

**Comment by Razharas** [comment]
Its your pr m8, i just said that there is a way for the resulting item to use the things from which it was created

**Pull request merged**